collectors/framework/models.py

```diff
@@ -570,3 +570,12 @@ def inner(*args, **kwargs):
         return inner
 
     return wrapper
+
+
+class KnownFeedEntry(models.Model):
```

Comment:
I do not like this approach as it is collector-specific (does not mean that I cannot be outvoted :smile:).

I would prefer creating a uniquely identifiable snippet instead (that is the reason why I was advocating for a snippet identifier). The following runs would look for the known scrape results among the snippets instead.

We could add a flag like **create_flaws** or something which could be set to **False** in the case that the snippet actually contains something we do not care for. The advantage would be that we would have a single place where to look for already know scrape results - both the ones resulting in flaws and the ones we ignored.

There would be a single storage for these, the **Snippet** model, and a single canonical way to identify them, **source:identifier** combination. No collector-specific functionality would be needed.

Reply:
I agree this will be the correct approach. It should work with or without the identifiers but I slightly favor having that field in there because it will be used for lookups heavily (each collector can still decide how to use that field).

Reply:
As already mentioned, this can just be an attribute in `Snippet.content`

Reply:
Done. Only `Snippet` model is used.